dts/bindings/kscan/focaltech,ft5336.yaml

```diff
@@ -11,3 +11,15 @@ properties:
     int-gpios:
       type: phandle-array
       required: false
+
+    power-gpio:
+      type: phandle-array
+      specifier-space: gpio
+      required: false
+      description: |
+        POWER pin.
+
+        Enable power for the circuitry of the touch-controller.
+        Boards like stm32l562e_dk needs it.
```

Comment:
Board-specific notes in generic bindings are a no-no, too :). Can you please just remove this line?

Also, heads-up that @jfischer-no has a PR going in parallel with yours which will cause conflicts for whoever gets merged second: https://github.com/zephyrproject-rtos/zephyr/pull/40765/

Reply:
Alright I'll remove that line. @jfischer-no 's PR looks more advanced and improves abstraction, it's ok to being merged after his PR, I'll adapt this one :)

Reply:
Thanks for your continued patience!